tests/test_git_push.py

```
@@ -0,0 +1,141 @@
+    _GIT_AUTHOR_NAME,
+    push_state,
+)
+
+
+# ---------------------------------------------------------------------------
+# Happy path
+# ---------------------------------------------------------------------------
+
+
+@patch("social_agent.git_push.subprocess.run")
+def test_push_state_success(mock_run: MagicMock, tmp_path: Path) -> None:
```

Ruff (0.15.0)

[warning] Unused function argument: `mock_run`

(ARG001)